app/views/roles/_past_role_holders.html.erb

```diff
@@ -0,0 +1,17 @@
+<% unless role.past_holders.empty? %>
+  <section id="past-role-holders" class="govuk-!-padding-bottom-9">
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "Past role holders:",
+    } %>
+
+    <%= render "components/taxon-list", {
+      items: role.past_holders.map { |rh|
```

Comment:
```suggestion
      items: role.past_holders.map do |rh|
        ...
      end
```

I think for longer blocks, it's better to use a `do/end`, as it's more readable.